Tests/OptimizelyTests-Common/ODPSegmentManagerTests copy.swift

```diff
@@ -0,0 +1,269 @@
+//
```

Comment:
This file looks like an accidental duplicate?

Reply:
@andrewleap-optimizely yes it is. Removed :)